aspnetcore/release-notes/aspnetcore-8.0.md

```diff
@@ -20,16 +20,205 @@ This article is under development and not complete. More information may be foun
+
+* Static server-side rendering to generate static HTML.
+* Interactive server rendering using the Blazor Server hosting model.
+* Interactive client rendering using the Blazor WebAssembly hosting model.
+* Automatic interactive client rendering using Blazor Server initially and then WebAssembly on subsequent visits after the Blazor bundle is downloaded and the .NET WebAssembly runtime activates. Automatic rendering usually provides the fastest app startup experience.
+
+Interactive render modes also prerender content by default.
+
+For more information, see <xref:blazor/components/render-modes?view=aspnetcore-8.0&preserve-view=true>.
+
+### New Blazor Web App template
+
+We've introduced a new Blazor project template: the *Blazor Web App* template. The new template provides a single starting point for using Blazor components to build any style of web UI. The template combines the strengths of the existing Blazor Server and Blazor WebAssembly hosting models with the new Blazor capabilities added in .NET 8: server-side rendering, streaming rendering, enhanced navigation and form handling, and the ability to add interactivity using either Blazor Server or Blazor WebAssembly on a per-component basis.
+
+As part of unifying the various Blazor hosting models into a single model in .NET 8, we're also consolidating the number of Blazor project templates. We removed the Blazor Server template, and the ASP.NET Core Hosted option has been removed from the Blazor WebAssembly template. Both of these scenarios are represented by options when using the Blazor Web App template.
+
+> [!NOTE]
+> Hosted Blazor WebAssembly remains supported in .NET 8 or later, and existing hosted Blazor WebAssembly apps can be upgraded to use .NET 8 or later API and features.
+
+For more information on the new Blazor Web App template, see the following articles:
+
+* <xref:blazor/tooling?view=aspnetcore-8.0&pivots=windows&preserve-view=true>
+* <xref:blazor/project-structure?view=aspnetcore-8.0&preserve-view=true>
+
+### Enhanced navigation and form handling
+
+With new form handling features for multiple forms on a page, a request is handled by the form with the matching form handler name with per-form model binding and form validation. Also in this release, standard HTML forms based on a plain HTML `<form>` element with server-side rendering is supported without using an `EditForm`.
+
+New antiforgery support is included in .NET 8. A new `AntiforgeryToken` component renders an antiforgery token as a hidden field, and the new `[RequireAntiforgeryToken]` attribute enables antiforgery protection. If an antiforgery check fails, a 400 (Bad Request) response is returned without form processing. The new antiforgery features are enabled by default for forms based on `Editform` and can be applied manually to standard HTML forms.
+
+For more information, see <xref:blazor/forms-and-input-components?view=aspnetcore-8.0&preserve-view=true>.
```

Comment:
This text is covering the static server rendering support for forms. Enhanced navigation and form handling is something different: it's a general enhancement to static server rendering where Blazor will intercept the browser request and do a fetch request instead. Blazor then receives the rendered response and patches it into the DOM, which avoids a page refresh and preserves more of the page state. Your app feels like it's behaving like a single-page app even though it's actually doing static server rendering.

```suggestion
### Enhanced navigation and form handling

Static server rendering typically performs a full page refresh whenever the user navigates to a new page or submits a form. In .NET 8, Blazor can enhance page navigations and form handling by intercepting the request and performing a fetch request instead. Blazor then handles the rendered response content by patching it into the browser DOM. Enhanced navigation and form handling avoids the need for a full page refresh and preserves more of the page state, so that pages load faster and more smoothly. Enhanced navigation and form handling is enabled by default for static server rendered pages when the blazor.web.js script is loaded.
```

Reply:
I applied this change.

I also kept what was here under a different heading, *Support for multiple forms and antiforgery*.